Source/ACE/Entity/WorldObject.cs

```diff
@@ -1460,7 +1460,8 @@ protected WorldObject(AceObject aceObject)
                 : this(new ObjectGuid(aceObject.AceObjectId))
         {
             AceObject = aceObject;
-
+            SetWeenieHeaderFlag();
+            SetWeenieHeaderFlag2();
```

Comment:
these shouldn't be needed, the WeenieHeaderFlags are set when an object reads the field. If its not updating, its a bug for sure